src/components/App.js

```diff
@@ -3,6 +3,9 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { Switch, Route, Redirect } from 'react-router';
 import classNames from 'classnames';
+import PrivateRoute from './auth/PrivateRoute';
```

Comment:
You imported PrivateRoute here but never used it in the routing below.
Also, I would suggest the AuthenticatedRoute or AuthRoute maybe?